doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -0,0 +1,79 @@
+.. _lsst.obs.decam.gen3:
+
+#################
+The Gen3 workflow
+#################
+
+These directions are for users who wish to create their own data repositories independent of any Rubin/LSST-supported shared repositories.
+
+Create a Gen3 data repository and write curated calibration products to it (these include crosstalk, linearity, and defects). The latter step creates the `DECam/calib` collection.
+
+.. prompt:: bash
+
+   cd /path/to/desired/repo
+   butler create .
+   butler register-instrument . lsst.obs.decam.DarkEnergyCamera
+   butler write-curated-calibrations . lsst.obs.decam.DarkEnergyCamera
+
+Ingest raw science and calibration (bias and flat) frames, which creates the `DECam/raw/all` collection. Run overscan correction on all the raws so they may be used as crosstalk sources during ISR.
+
+This example assumes the user has setup the `~lsst.cp.pipe`_ module, which is needed for constructing calibration products from raws.
```

Comment:
I think any example like this should start with `setup lsst_distrib` as step 0 (and for the gen2 section below), so that the user has everything necessary?

Reply:
I don't see `setup lsst_distrib` anywhere in the doc yet?

Reply:
These lines are outdated; there's a mention of `lsst_distrib` now on line 7, which I guess should be 2 lines, because sentences.